operator/v1alpha1/types.go

```diff
@@ -150,3 +150,31 @@ type DelegatedAuthorization struct {
 	// disabled indicates that authorization should be disabled.  By default it will use delegated authorization.
 	Disabled bool `json:"disabled,omitempty"`
 }
+
+// StaticPodOperatorStatus is status for controllers that manage static pods.  There are different needs because individual
+// node status must be tracked.
+type StaticPodOperatorStatus struct {
```

Comment:
What is this when OperatorStatus moves to ClusterOperator?

Reply:
> What is this when OperatorStatus moves to ClusterOperator?

This type is embedded inside of our operator resource.  See https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/apis/kubeapiserver/v1alpha1/types.go#L50 today.  I don't think this has to change.  It is not served.